src/transforms/RenameTypes.ts

```diff
@@ -98,14 +98,24 @@ export default class RenameTypes implements Transform {
       const newObject = Object.create(Object.getPrototypeOf(value));
       let returnNewObject = false;
 
-      Object.keys(value).forEach(key => {
-        const oldChild = value[key];
-        const newChild = this.renameTypes(oldChild, key);
-        newObject[key] = newChild;
-        if (newChild !== oldChild) {
-          returnNewObject = true;
-        }
-      });
+      if (newObject instanceof Array) {
```

Comment:
I was going to say this should be `if (Array.isArray(newObject)) {`, but that doesn't actually work given the `Object.create(Object.getPrototypeOf(value))` pattern above, which makes me think we shouldn't be allowing these pseudo-arrays to escape this function at all.